src/firebase.js

```diff
@@ -0,0 +1,50 @@
+import { initializeApp } from "firebase/app";
+import * as mess from "firebase/messaging";
+// import { getMessaging, getToken, onMessage } from "firebase/messaging";
+
+var firebaseConfig = {
```

Comment:
Firebase config will pull fields from .env file. We should move it there - will help in making this generic.

Reply:
![image](https://user-images.githubusercontent.com/76895787/170893259-0f200519-7ed4-4b5c-9f4e-9049c598ad2d.png)

Did it. Just please confirm if this will work

Reply:
Made the necessary changes in `.env` file too

![image](https://user-images.githubusercontent.com/76895787/170893283-77dc6c32-aa13-4db6-ba03-f9e16902dfed.png)

Reply:
Also, don't expose the env variables.

Reply:
How can I do that